### PR TITLE
Update README.md file

### PR DESCRIPTION
**Pull request by FareheSoheil**
Fixed 

<pre>devtools::tests() </pre>

to 

<pre>devtools::test() </pre>

and added a small guide for running the tests.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,10 +43,10 @@ The following packages should be installed: `Rcpp`, `RcppEigen`, `BH`, `devtools
 Optionally the user can run tests and examples by running (from an `R` terminal):
 
 ```R
-devtools::tests()
+devtools::test()
 devtools::run_examples()
 ```
-
+For running the tests, user must be in the /tests directory.
```
Comment (vfisikop) [inline README.md]: This is not needed.

**Review by vfisikop — COMMENTED** [review]
Thanks for this PR.

File: README.md (hunk 1 above)
Comment (FareheSoheil) [reply README.md]: Removed. I have created a new pull request.

**Comment by FareheSoheil** [comment]
Extra explanation removed.

**Comment by vissarion** [comment]
Thanks! Please do not create more than one PR for the same topic!

**Pull request merged**